lang/string_extractor/parsers/difficulty_impact.py

```diff
@@ -0,0 +1,6 @@
+from ..write_text import write_text
+
+
+def parse_difficulty_impact(json, origin):
+    write_text(json["name"], origin,
+               comment="Aspect of gameplay affected by difficulty")
```

Comment:
Since you have a comment field in `difficulty_impact` JSON data, I suggest including that in translation comments to give translators better context.

```suggestion
    comment = ["Aspect of gameplay affected by difficulty"]
    if "//" in json:
        comment.append(json["//"])
    write_text(json["name"], origin, comment=comment)
```

Reply:
Neat! I'll do that from now on.